concordium-node/src/configuration.rs

```diff
@@ -380,7 +380,15 @@ pub struct GRPC2Config {
         env = "CONCORDIUM_NODE_GRPC2_ENDPOINT_CONFIG",
         requires = "grpc2-listen-addr"
     )]
-    pub endpoint_config:  Option<PathBuf>,
+    pub endpoint_config:            Option<PathBuf>,
+    #[structopt(
+        long = "grpc2-health-max-finalized-delay",
+        help = "Maximum amount of seconds that the time of the last finalized block can be behind \
+                present before the health check fails.",
+        env = "CONCORDIUM_NODE_GRPC2_HEALTH_MAX_FINALIZED_DELAY",
+        default_value = "300"
```

Comment:
Is this a sensible default? 5 min seems too much?

Reply:
I think it is somewhat sensible. 2min will have a reasonable amount of false positives which is not ideal on mainnet.

Maybe 3min would be good enough. Don't know, but that is also why I made it a configuration option since we have to experiment a bit.

Reply:
So I will keep the default as 5min for now.